.github/workflows/ci.yml

```diff
@@ -12,33 +12,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.4', '2.5', '2.6', '2.7', '3.0', jruby-head, truffleruby-head]
-        redis: ['4']
-        search: [['opensearch-ruby:2.1.0', 'opensearchproject/opensearch:2.2.1']]
-        include:
-          # Redis 3
-          - ruby: '2.7'
-            redis: '3'
-            search: ['opensearch-ruby:2.1.0', 'opensearchproject/opensearch:2.2.1']
-          # Opensearch 1.0
-          - ruby: '2.7'
-            redis: '4'
-            search: ['opensearch-ruby:1.0.1', 'opensearchproject/opensearch:1.0.1']
-          # Elasticsearch 7.13
-          - ruby: '2.7'
-            redis: '4'
-            search: ['elasticsearch:7.13.3', 'elasticsearch:7.13.4']
-          # Redis 5
-          - ruby: '2.7'
-            redis: '5'
-            search: ['opensearch-ruby:2.1.0', 'opensearchproject/opensearch:2.2.1']
-          # Ruby 2.3 & Elasticsearch 7.5
-          - ruby: '2.3'
-            redis: '4'
-            search: ['elasticsearch:7.5.0', 'elasticsearch:7.13.4']
+        ruby: ['2.3', '2.4', '2.5', '2.6', '2.7', '3.0', '3.1', '3.2', '3.3', '3.4', head, jruby-head, truffleruby-head]
+        redis: ['3', '4', '5', '6', '7', '8']
+        search: [
+          ['opensearch-ruby:1', 'opensearchproject/opensearch:1'],
+          ['opensearch-ruby:2', 'opensearchproject/opensearch:2'],
+          ['opensearch-ruby:3', 'opensearchproject/opensearch:3'],
+          ['elasticsearch:7', 'elasticsearch:7.17.28'],
+          ['elasticsearch:8', 'elasticsearch:8.18.2'],
+          ['elasticsearch:9', 'elasticsearch:9.0.2']
+        ]
     services:
       redis:
-        image: redis
+        image: redis:${{ matrix.redis }}
```

Comment:
`matrix.redis` is actually the gem version, not the server version.

Reply:
Ah yes I see that now, that simplifies things a bunch if we just want to use the latest server version.